server/generate/srcfiles.go

```diff
@@ -90,10 +90,62 @@ var (
 		"shell/pty/pty_unsupported.go",
 
 		"pivots/pivots.go",
-		"pivots/namedpipe/namedpipe.go",
-		"pivots/namedpipe/npipe_windows.go",
-		"pivots/namedpipe/znpipe_windows_386.go",
-		"pivots/namedpipe/znpipe_windows_amd64.go",
+		"pivots/winio/reparse.go",
+		"pivots/winio/tools/etw-provider-gen/main.go",
+		"pivots/winio/backup.go",
+		"pivots/winio/archive/tar/stat_atim.go",
+		"pivots/winio/archive/tar/stat_atimespec.go",
+		"pivots/winio/archive/tar/writer.go",
+		"pivots/winio/archive/tar/common.go",
+		"pivots/winio/archive/tar/stat_unix.go",
+		"pivots/winio/archive/tar/reader.go",
+		"pivots/winio/hvsock.go",
+		"pivots/winio/privilege.go",
+		"pivots/winio/syscall.go",
+		"pivots/winio/backuptar/tar.go",
+		"pivots/winio/backuptar/noop.go",
+		"pivots/winio/zsyscall_windows.go",
+		"pivots/winio/file.go",
+		"pivots/winio/wim/decompress.go",
+		"pivots/winio/wim/validate/validate.go",
+		"pivots/winio/wim/lzx/lzx.go",
```

Comment:
`wim` has not been synced to the source tree. I'm not sure whether or not it was intentional, but in any case, if we need to use it for another feature, it could be nice to sync it (but not include it in the `srcfiles` list).

Reply:
done